packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -575,25 +572,28 @@ contract CrabStrategyV2 is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Own
      * @dev check the signer and swap tokens in the order
      * @param _remainingAmount quantity the manager wants to sell
      * @param _clearingPrice the price at which the manager is buying
-     * @param _orderPrice the selling price of each order
      * @param _order a signed order to swap tokens
      */
     function _execOrder(
         uint256 _remainingAmount,
         uint256 _clearingPrice,
-        uint256 _orderPrice,
         Order memory _order
     ) internal {
-        require(_clearingPrice >= _orderPrice, "Clearing Price should be at least Seller Price");
+        // Check order beats clearing price
+        if (_order.isBuying){
```

Comment:
isBuying is from perspective of market maker, right?

Reply:
Yep